packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -3482,7 +3489,37 @@ export default class Meeting extends StatelessWebexPlugin {
       // for multistream WCME automatically un-publishes the old track when we publish a new one
       await this.unpublishTrack(oldTrack);
     }
-    await this.publishTrack(this.mediaProperties.shareTrack);
+    await this.publishTrack(this.mediaProperties.shareVideoTrack);
+  }
+
+  /**
+   * Stores the reference to a new screen share audio track, sets up the required event listeners
+   * on it, cleans up previous track, etc.
+   * It also sends the floor grant/release request.
```

Comment:
this part of the comment is not true, the function doesn't send the floor grant/release request. I realize you copied this from the existing setLocalShareVideoTrack() comments - they're incorrect too, can you fix them as well, please?

Reply:
oops, fixed both